lib/js/src/compose/types/module.ts

```diff
@@ -164,9 +168,15 @@ export class Module {
 
     recordDeDup: {
       // Always true for now since empty array of rules is the same as disabling it
-      enabled: true,
-      strict: false,
-      rules: [],
+      rules: [
+        {
```

Comment:
Same question as above, is this the only rule? Do we need extra ones?

Reply:
Yes, from Vivek

Reply:
rules logic is not updated